ee/cis/macos-15/cis-policy-queries.yml

```diff
@@ -3464,3 +3464,28 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1
   contributors: sharon-fdm
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Home Folders Are Secure
```

Comment:
@defensivedepth, for some reason I see two copies of this policy.
[here](https://github.com/fleetdm/fleet/pull/24677/files#diff-5c61e31d49a9cf30feddbe1cc773890897b2091e96521f90735f8d5f406ec735R2511)